From 65a635d3d80afc301e1d6a9b528c61fe24af7e08 Mon Sep 17 00:00:00 2001
From: Peter Hutterer <peter.hutterer@who-t.net>
Date: Tue, 5 Apr 2016 10:49:41 +1000
Subject: [PATCH libwacom 2/2] data: make all buttons on the EKR 'left' buttons

Left/right in the data file meaning refers to "left of the tablet area" which
doesn't apply for the EKR. Move them all to the left so userspace doesn't get
confused.

Signed-off-by: Peter Hutterer <peter.hutterer@who-t.net>
---
 data/ek-remote.tablet | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/data/ek-remote.tablet b/data/ek-remote.tablet
index a793bf4..9e3f5e1 100644
--- a/data/ek-remote.tablet
+++ b/data/ek-remote.tablet
@@ -24,8 +24,7 @@ Buttons=18
 #StatusLEDs=Ring
 
 [Buttons]
-Left=C;A;B;F;G;J;M;N;P;Q
-Right=D;E;I;L;H;K;O;R
+Left=C;A;B;F;G;J;M;N;P;Q;D;E;I;L;H;K;O;R
 
 #Note that though this device has 3 ring modes
 #this mode selection does not function the same
-- 
2.5.5

